client/src/hooks/helpers/useManageTransactionsPending.ts

```diff
@@ -94,14 +94,20 @@ export default function useManageTransactionsPending(): void {
      * refetches are triggered and blockNumberWithLatestTx to null.
      */
     if (blockNumber && blockNumberWithLatestTx && blockNumber > blockNumberWithLatestTx) {
-      refetchAvailableFundsGlm();
-      refetchDeposit();
-      refetchEstimatedEffectiveDeposit();
-      refetchLockedSummaryLatest();
-      refetchWithdrawals();
+      const lastTransaction = transactionsPending?.at(0);
+
+      if (lastTransaction?.type === 'withdrawal') {
+        refetchWithdrawals();
+        refetchAvailableFundsGlm();
```

Comment:
Why withdrawing should result in refetching of GLMs available in the wallet? Rather, it should be a refetch from `useAvailableFundsEth`, which is not here at all, but it should. Can you please adjust that?

Reply:
After check only `refetchWithdrawals` is required in this case.